lib/dispatch.go

```diff
@@ -249,6 +250,26 @@ type callable struct {
 	DenyRPC   bool
 }
 
+// AllMethods returns a method set for documentation purposes,
+func (inst *Instance) AllMethods() []MethodSet {
```

Comment:
this method being used _only_ for documentation is a potential source of drift. I'd initially intended to combine this with the function body of `*lib.Instance.RegisterMethods` somehow, but haven't gotten to it. Can we complete this comment & add a `TODO`?